docs/docs-beta/docs/dagster-plus/insights/asset-metadata.md

```diff
@@ -4,4 +4,58 @@ sidebar_label: "Integrate asset metadata"
 sidebar_position: 1
 ---
 
-# Integrate asset metadata into Dagster+ Insights
\ No newline at end of file
+Out of the box, Dagster+ Insights gives you visibility into a variety of common metrics across your data platform.
+By creating custom metrics from asset metadata, you can use Insights to perform historical aggregation on any
+data your assets can emit.
+
+## What you'll learn
+
+- How to add asset metadata as a custom metric to Dagster+ Insights
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need a Dagster+ account on the Pro plan.
+
+</details>
+
+## Step 1: Emit numeric metadata on your assets at runtime
+
+You'll need one or more assets that emit the same metadata key at run time. Insights metrics
+are most valuable when you have multiple assets that emit the same kind of metadata. Follow [the metadata
+guide](/guides/data-modeling/metadata#attaching-metadata-to-an-asset-at-runtime) to add numeric metadata
+to your asset materializations.
+
+## Step 2: Enable viewing numeric metadata in Dagster+ Insights
+
+Once your assets are emitting numeric metadata values, you'll be able to enable viewing them in the Insights UI.
+
+To add your metadata key to the list of metrics shown in Insights, click **Edit** in the sidebar next to the **User provided metrics** header:
+
+![Viewing the Insights tab in the Dagster+ UI](/img/placeholder.svg)
+{/* <Image
+alt="Viewing the Insights tab in the Dagster UI"
+src="/images/dagster-cloud/insights/insights-tab.png"
+width={2640}
+height={1516}
+*/}
+
+In the dialog that appears, use the eye indicator to show or hide metrics in Insights. Selected metrics will be immediately available to view in Insights in the sidebar and on individual asset pages.
```

Comment:
I found mention of "immediately available" followed by a note on 24 hours to be contradicting.
```suggestion
In the dialog that appears, use the eye indicator to show or hide metrics in Insights. Selected metrics will be visible in both the Insights sidebar and on individual asset pages.
```

Reply:
+1